File: BAEKJOON/Javascript/19000/19500/19532.js
/*
Title : 수학은 비대면강의입니다
Link : https://www.acmicpc.net/problem/19532
*/

const fs = require("fs");
const filepath =
  process.platform === "linux" ? "/dev/stdin" : __dirname + "/input.txt";
const input = fs.readFileSync(filepath).toString().trim();

const [a, b, c, d, e, f] = input.split(" ").map(Number);
const x = (c * e - b * f) / (a * e - b * d);
const y = (a * f - c * d) / (a * e - b * d);
console.log(`${parseInt(x)} ${parseInt(y)}`);

File: BAEKJOON/Python/19000/19500/19532.py
"""
Title : 수학은 비대면강의입니다
Link : https://www.acmicpc.net/problem/19532
"""

from sys import stdin

input = stdin.readline


if __name__ == "__main__":
    a, b, c, d, e, f = map(int, input().split())
    x = (c * e - b * f) / (a * e - b * d)
    y = (a * f - c * d) / (a * e - b * d)
    print(int(x), int(y))
